packages/studioCMS/src/integrations/studioCMSDashboard/routes/dashboard/partials/LivePreview.astro

```diff
@@ -0,0 +1,10 @@
+---
+// export const partial = true; - Disabled due to regression | See @ https://github.com/withastro/astro/issues/11103
```

Comment:
Any updates on this regression?

Reply:
Looks like it has been resolved, uncommented in [e146d7e](https://github.com/astrolicious/studiocms/pull/77/commits/e146d7ede40aea7e05f8622f2c08e901e36f7f93)